scripts/actionlint.sh

```diff
@@ -3,3 +3,22 @@
 set -euo pipefail
 
 go run github.com/rhysd/actionlint/cmd/actionlint@v1.7.1 "${@}"
+
+echo "Checking use of scripts/* in GitHub Actions workflows..."
+SCRIPT_USAGE=
+for file in .github/workflows/*.{yml,yaml}; do
+  # Skip if no matches found (in case one of the extensions doesn't exist)
+  [[ -f "$file" ]] || continue
+
+  # Search for scripts/* except for scripts/run_task.sh
+  MATCHES=$(grep -H -n -P "scripts/(?!run_task\.sh)" "$file" || true)
```

Comment:
so this essentially ensures we only use `scripts/run_task.sh` and nothing else in `scripts`?